redis-enterprise-cluster.yaml

```diff
@@ -19,5 +19,5 @@ spec:
   redisEnterpriseImageSpec:
     imagePullPolicy:  IfNotPresent
     Repository:       redislabs/redis
-    versionTag:       5.2.0-14
+    versionTag:       5.2.2-22
```

Comment:
@danni-m - any reason of keeping specific image here?

Reply:
@michalrabinowitch the reasoning was mostly to be explicit and to easily see which RS version is being deployed/